modules/lvms-limitations-to-configure-size-of-devices.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="limitations-to-configure-size-of-devices_{context}"]
+= Limitations to configure the size of the devices to be used in LVM Storage
+
+The limitations to configure the size of the devices that you can use to provision storage using {lvms} are as follows:
+
+* The total storage size that you can provision is limited by the size of the underlying Logical Volume Manager (LVM) thin pool and the over-provisioning factor.
+* The size of the logical volume depends on the size of the Physical Extent (PE) and the Logical Extent (LE).
+** You can define the size of PE and LE during the physical and logical device creation.
+** The default PE and LE size is 4 MB.
+** If the size of the PE is increased, the maximum size of the LVM is determined by the kernel limits and your disk space.
+
+.Size limits for different architectures using the default PE and LE size
+[cols="1,1,1,1,1", width="100%", options="header"]
+|====
+|Architecture
+|RHEL 6
+|RHEL 7
+|RHEL 8
+|RHEL 9
+
+|32-bit
+|16 TB
+|-
+|-
+|-
+
+|64-bit
+
+|8 EB ^[1]^
+
+100 TB ^[2]^
+|8 EB ^[1]^
+
+500 TB ^[2]^
```

Comment:
```suggestion
|500 TB ^[2]^
```

Reply:
Looks like these are rendering ok in the table, but I think they need a |. Wdyt?

Reply:
These are two values in the same row. They are kept this way intentionally and the table is rendered as follows:

![Screenshot from 2024-03-19 10-31-37](https://github.com/openshift/openshift-docs/assets/148952239/fe274774-1bc4-4a36-807c-7c290b895e5b)

[1] for Theoretical size
[2] for Tested size